unsafe/src/main/java/org/apache/spark/unsafe/types/UTF8String.java

```diff
@@ -77,6 +78,15 @@ public static UTF8String fromString(String str) {
     }
   }
 
+  /**
+   * Creates an UTF8String that contains `length` spaces.
+   */
+  public static UTF8String blankString(int length) {
+    byte[] spaces = new byte[length];
+    Arrays.fill(spaces, (byte) ' ');
```

Comment:
Char implements UTF-16 (http://docs.oracle.com/javase/7/docs/api/java/lang/Character.html)

"The char data type (and therefore the value that a Character object encapsulates) are based on the original Unicode specification, which defined characters as fixed-width 16-bit entities".

Shall I still change it?